.github/workflows/test.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    tags:
+      - dev
+env:
+  TF_VERSION: 1.2.4
```

Comment:
remove TF Version

Reply:
> LG, just few qns.
> 
> Pls clarify what you meant by moving backend one folder up :)

Oops! I un-did this to avoid an even bigger diff. Also, I wasn't sure if the zipped prod build referenced the current file path so I thought it would be better to leave it as is.